vignettes/social_smell_showcase.Rmd

```diff
@@ -32,6 +32,9 @@ The bottom line is, the required effort to obtain the mailing list data will var
 # Libraries
 
 Please ensure the following R packages are installed on your computer. 
+On Ubuntu the libfontconfig1-dev package and libharfbuzz-dev package are required. It not, here are some command recommended as the following:
```

Comment:
This is very helpful for the wiki on Troubleshooting Dependencies. Could you add to it instead of the notebook?